charts/kyverno/values.yaml

```diff
@@ -49,22 +49,30 @@ podSecurityContext: {}
 priorityClassName: ""
 
 antiAffinity:
-  # This can be disabled in a 1 node cluster.
-  enable: true
-
-affinity:
-  podAntiAffinity:
-    preferredDuringSchedulingIgnoredDuringExecution:
+  preferredDuringSchedulingIgnoredDuringExecution:
     - weight: 1
       podAffinityTerm:
         labelSelector:
           matchExpressions:
-          - key: app.kubernetes.io/name
-            operator: In
-            values:
-            - kyverno
+            - key: app.kubernetes.io/name
+              operator: In
+              values:
+                - kyverno
         topologyKey: kubernetes.io/hostname
 
+podAffinity: {}
+
+nodeAffinity: {}
+
+affinity:
+  podAntiAffinity:
+    # This can be disabled in a 1 node cluster.
```

Comment:
Just to clarify one thing, this doesn't need to be disabled in single node clusters because the anti-affinity uses `preferredDuringSchedulingIgnoredDuringExecution` so it's more of a "suggestion" for anti-affinity than hard requirement.

Reply:
maybe the comment lead in the wrong direction. 
We could change it to: "using  this option will schedule pods on different nodes if possible"

Reply:
That sounds reasonable.

Reply:
done